lib/resque.rb

```diff
@@ -147,15 +167,25 @@ def inline=(inline)
   #
   # Returns nothing
   def push(queue, item)
-    watch_queue(queue)
-    redis.rpush "queue:#{queue}", encode(item)
+    redis.pipelined do
```

Comment:
Same behavior as [here](https://github.com/al2o3cr/resque/pull/2/files#diff-102541f6f2ba476fceb8e45917e63420R96) in `push_to_queue`